### PR TITLE
fixing remove

### PR DESCRIPTION
**Pull request by endophage**
Signed-off-by: David Lawrence <david.lawrence@docker.com> (github: endophage)

### CODE REVIEW

**Comment by NathanMcCauley** [comment]
needs tests

File: client/helpers.go [hunk 1]
```diff
@@ -26,13 +27,16 @@ func getRemoteStore(baseURL, gun string, rt http.RoundTripper) (store.RemoteStor
...
-		if err != nil {
-			return err
+		switch c.Scope() {
+		case changelist.ScopeTargets:
+			err := applyTargetsChange(repo, c)
+			if err != nil {
+				return err
+			}
+		default:
+			logrus.Debug("scope not supported: ", c.Scope())
```
Comment (diogomonica) [inline client/helpers.go]: This is a change in behavior. Now we're not returning an error here.

File: cmd/notary/tuf.go [hunk 2]
```diff
@@ -17,7 +17,7 @@ import (
 )
 
 // FIXME: This should not be hardcoded
-const hardcodedBaseURL = "https://notary-server:4443"
+const hardcodedBaseURL = "http://notary-server:4443"
```
Comment (diogomonica) [inline cmd/notary/tuf.go]: remove!

File: client/helpers.go (hunk 1 above)
Comment (endophage) [inline client/helpers.go]: We weren't before either. Unsupported changes should never be in the changelist so we skip them.

**Comment by endophage** [comment]
@NathanMcCauley I would swear you have a bot that adds "needs tests" to every PR :-P

**Comment by diogomonica** [comment]
@endophage aren't you Mr. 80%+ test coverage?

**Comment by endophage** [comment]
@diogomonica  Yes, but "Demo Early"

**Pull request merged**